### PR TITLE
add baselining to videos on fronts test

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

Adding the baselining to see how long we would have to run a test with videos on fronts to make a statistical difference.
## What is the value of this and can you measure success?

We see numbers go ⬆️ .
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@harrysalmon @akash1810

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/play-video-on-fronts.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            return config.page.isFront && qwery('video:not([data-embed-path])').length > 0;
+        };
+
+        this.variants = [
+            {
+                id: 'baseline1',
+                test: function () {}
+            },
+            {
+                id: 'baseline2',
```
Comment (akash1810) [inline static/src/javascripts/projects/common/modules/experiments/tests/play-video-on-fronts.js]: why two variants?

**Comment by akash1810** [comment]
I don't quite understand how this test gets used... to my mind, if you're in the test, the DOM wouldn't have  a `video` element, or at least it would be wrapped around an `a` that links to the article.

**Comment by jamesgorrie** [comment]
@akash1810 This is to baseline the test. Basically we are seeing how many people fall into the `canRun`, and then we can work out a what % we will need to test against to get a statistically significant amount of data.

In short - this test does nothing yet but count for us.

I have also updated this to [use](https://github.com/guardian/frontend/pull/12963/files#diff-de94f74285654768f5cff0dfc33b9ec6R23) my [other PR](https://github.com/guardian/frontend/pull/12965) - so this test will only run if there is a main-media article on a front.

**Comment by akash1810** [comment]
👍

**Comment by harrysalmon** [comment]
Great!

On 18 May 2016 at 15:59, James Gorrie <notifications@github.com> wrote:

> Merged #12963 <https://github.com/guardian/frontend/pull/12963>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/12963#event-664527245>
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Pull request merged**